From e7121de237a1873c3241a5b8451e7d00a3d41524 Mon Sep 17 00:00:00 2001
From: Akira TAGOH <akira@tagoh.org>
Date: Fri, 3 Oct 2014 12:26:42 +0900
Subject: Revert "Bug 73291 - poppler does not show fl ligature"

This reverts commit c6aa4d4bfcbed14f39d070fe7ef90a4b74642ee7.

This issue has been fixed in poppler and we no longer need to patch it out in fontconfig.

diff --git a/conf.d/30-metric-aliases.conf b/conf.d/30-metric-aliases.conf
index 08c8ba3..49a9602 100644
--- a/conf.d/30-metric-aliases.conf
+++ b/conf.d/30-metric-aliases.conf
@@ -77,15 +77,12 @@ but in an order preferring similar designs first.  We do this in three steps:
 	  </default>
 	</alias>
 
-<!--
-     Due to Bug#73291, commented out those lines until the broken font are fixed.
 	<alias binding="same">
 	  <family>TeX Gyre Termes</family>
 	  <default>
 	  <family>Times</family>
 	  </default>
 	</alias>
--->
 
 	<alias binding="same">
 	  <family>Nimbus Mono L</family>
@@ -418,10 +415,7 @@ but in an order preferring similar designs first.  We do this in three steps:
 	<alias binding="same">
 	  <family>Times</family>
 	  <accept>
-<!--
-     Due to Bug#73291, commented out this line until the broken font are fixed.
 	  <family>TeX Gyre Termes</family>
--->
 	  <family>Nimbus Roman No9 L</family>
 	  </accept>
 	</alias>
-- 
cgit v0.10.2

